e2e/specs/invalid.vitals.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test } from '../core';
+import { expect } from '@playwright/test';
+import { type Visit } from '@openmrs/esm-framework';
+import { generateRandomPatient, type Patient, startVisit, endVisit } from '../commands';
+import { BiometricsAndVitalsPage } from '../pages';
+
+let patient: Patient;
+let visit: Visit;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+  visit = await startVisit(api, patient.uuid);
+});
+
+test('Flagging abnormal vitals in the Patient Chart', async ({ page }) => {
+  const biometricsPage = new BiometricsAndVitalsPage(page);
+
+  await test.step('Given a patient record exists and the user is on the "Patient Chart" page', async () => {
+    await biometricsPage.goTo(patient.uuid);
+  });
+
+  await test.step('When the user navigates to the "Vitals & Biometrics" section', async () => {
+    await biometricsPage.goTo(patient.uuid);
+  });
+
+  await test.step('And clicks on "Add New Vital/Biometric"', async () => {
+    await biometricsPage.page.getByText(/record biometrics/i).click();
+  });
+
+  await test.step('Then a form to enter vitals/biometrics details should be displayed', async () => {
+    await expect(biometricsPage.page.getByText(/record vitals and biometrics/i)).toBeVisible();
+  });
+
+  await test.step('When the user enters abnormal vitals/biometric values', async () => {
```

Comment:
Is the test for vitals or biometrics?

Reply:
vitals @kdaud

Reply:
Shall we break down each value into sub steps? Generic steps can be confusing since they don't show what information was filled in the form.